### ISSUE TITLE
lpa loss

### ISSUE THREAD

**Issue by qqaazxddrr** [Vimalakirti/GCN-LPA-PyTorch#1]
loss_lpa = F.nll_loss(y_hat, labels)

Here you should use the labels of the training set.
